src/Sarif/Core/StackFrame.cs

```diff
@@ -55,10 +55,7 @@ public static StackFrame Create(System.Diagnostics.StackFrame dotNetStackFrame)
 
             if (nativeOffset != -1)
             {
-                stackFrame.Properties = new Dictionary<string, string>
-                {
-                    { "NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture) }
-                };
+                stackFrame.SetProperty("NativeOffset", nativeOffset.ToString(CultureInfo.InvariantCulture));
```

Comment:
Yes, that's next.

---
In reply to: [62771090](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62771090) [](ancestors = 62771090,62750080)